kubernetes/environments/kind/api/provisioning.yaml

```
@@ -0,0 +1,47 @@
+impactTypes:
+  - displayname: Performance Degradation
+  - displayname: Connectivity Problems
+  - displayname: Maintenance
+  - displayname: Unknown
+
+components:
+  - displayname: gx-scs
+    labels:
+      provider: plusserver
+      technology: openstack
+  - displayname: Website
+    labels: {}
+  - displayname: Jitsi
+    labels: {}
+  - displayname: Hedgedoc
+    labels: {}
+  - displayname: Moin Cluster
+    labels: {}
+  - displayname: Harbor
+    labels:
+      system: gx-scs
+  - displayname: Grafana
+    labels:
+      app: observability-cluster
+  - displayname: Prometheus
+    labels:
+      app: observability-cluster
+  - displayname: Zuul
+    labels:
+      system: gx-scs
+
```

Nextcloud should be included
```suggestion
  - displayname: Nextcloud
    labels: {}
```

---

IIRC, Nextcloud runs on an UCS installation, which also serves as IdP for Harbor.

So maybe also add UCS as IdP, here.